src/components/Tags.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
```

Comment:
not sure about creating tags comp. u pulled list as third party so u need just tag component which will be button and will return in one way or another its id when clicked. or we don't need to create any new component if this construction is used on home page only